plugins/scaffolder/src/components/fields/OwnershipEntityRefPicker/OwnershipEntityRefPicker.tsx

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { identityApiRef, useApi } from '@backstage/core-plugin-api';
+import { TextField, FormControl } from '@material-ui/core';
+import {
+  OwnershipEntityRefPickerProps,
+  OwnershipEntityRefPickerSchema,
+} from './schema';
+import { Autocomplete } from '@material-ui/lab';
+import { catalogApiRef } from '@backstage/plugin-catalog-react';
+import { NotFoundError } from '@backstage/errors';
+
+export { OwnershipEntityRefPickerSchema };
+
+export const OwnershipEntityRefPicker = (
+  props: OwnershipEntityRefPickerProps,
+) => {
+  const {
+    schema: { title, description },
+    required,
+    rawErrors,
+  } = props;
+
+  const identityApi = useApi(identityApiRef);
+  const catalogApi = useApi(catalogApiRef);
+  const [groups, setGroups] = useState<string[]>([]);
+  const [selectedGroup, setSelectedGroup] = useState<string | null>(null);
+
+  useEffect(() => {
+    const fetchUserGroups = async () => {
+      const identity = await identityApi.getBackstageIdentity();
+      const userIdentity = identity.ownershipEntityRefs;
+
+      if (!userIdentity) {
+        throw new NotFoundError('No ownership entity refs found');
```

Comment:
Maybe use the `alertApi` here to post an error message rather than throwing?

Reply:
Did not know this was a thing, TIL!

Thanks!